packages/zoe/src/zoeService/zoe.js

```diff
@@ -62,6 +62,17 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
     return installation;
   };
 
+  /**
+   * @type {GetAmountOfInvitationThen}
+   */
+  const getAmountOfInvitationThen = async (invitationP, onFulfilled) => {
+    const onRejected = () =>
+      assert.fail(details`A Zoe invitation is required, not ${invitationP}`);
+    return E(invitationKit.issuer)
+      .getAmountOf(invitationP)
+      .then(onFulfilled, onRejected);
```

Comment:
I like the way this pulls out `onRejected`. It's the same number of lines as I used, but it's much more readable.